tree/treeplayer/inc/ROOT/TDFHistoModels.hxx

```diff
@@ -30,12 +30,15 @@ struct TH1DModel {
    int fNbinsX = 128;
    double fXLow = 0.;
    double fXUp = 64.;
+   std::vector<double> fBinXEdges;
 
    TH1DModel() = default;
    TH1DModel(const TH1DModel &) = default;
    ~TH1DModel();
    TH1DModel(const ::TH1D &h);
    TH1DModel(const char *name, const char *title, int nbinsx, double xlow, double xup);
+   TH1DModel(const char *name, const char *title, Int_t nbinsx, const float *xbins);
```

Comment:
Any specific reason why `nbinsx` is `Int_t` here but `int` above?

Reply:
No. It was a mistake in the first implementation. Already fixed on my local branch :)